pagopa/pn-frontend
# Create notification with sender denomination too long - pn-10623

## Short description
Managed the case of long sender denomination in the new notification form

## List of changes proposed in this pull request
- Added new field in the first step of notification creation
- Added new test case

## How to test
Login with PA ->  go to notification creation -> check that everything is ok
Login with PA -> change the user saved in local storage with a long organization.name field -> reload the page -> go to notification creation -> check that the sender denomination field is editable and error is shown -> edit the field, go ahead and go back, and check that the field is still editable but no error is shown -> continue to the last step and check that the request sent to the be has the new value
